src/main/java/roomescape/service/ReservationAdminService.java

```diff
@@ -1,28 +1,88 @@
 package roomescape.service;
 
 import org.springframework.stereotype.Service;
-import roomescape.model.Reservation;
-import roomescape.model.ReservationCreateDto;
-import roomescape.model.ReservationCreateResponseDto;
-import roomescape.model.ReservationTime;
+import roomescape.exception.ReservationDateInvalidException;
+import roomescape.exception.ReservationDatePastException;
+import roomescape.exception.ReservationDuplicateException;
+import roomescape.exception.ReservationNameInvalidException;
+import roomescape.model.*;
 import roomescape.respository.ReservationDAO;
 import roomescape.respository.ReservationTimeDAO;
+import roomescape.respository.ThemeDAO;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
 import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
 
 @Service
 public class ReservationAdminService {
     private final ReservationDAO reservationDAO;
     private final ReservationTimeDAO reservationTimeDAO;
+    private final ThemeDAO themeDAO;
 
-    public ReservationAdminService(ReservationDAO reservationDAO, ReservationTimeDAO reservationTimeDAO) {
+    public ReservationAdminService(ReservationDAO reservationDAO, ReservationTimeDAO reservationTimeDAO,
+                                   ThemeDAO themeDAO) {
```

Comment:
```suggestion
    public ReservationAdminService(ReservationDAO reservationDAO,
                                   ReservationTimeDAO reservationTimeDAO,
                                   ThemeDAO themeDAO) {
```
이렇게 적어주면 파라미터를 확인하기 더 편할 것 같아요.